src/instance.ts

```diff
@@ -41,6 +61,15 @@ import {Database} from './database';
  * const instance = spanner.instance('my-instance');
  */
 class Instance extends common.ServiceObject {
+  // tslint:disable-next-line: no-any
+  metadata: any;
+  formattedName_: string;
+  // tslint:disable-next-line: no-any
+  request: any;
```

Comment:
Can we get/make a proper type for this and `requestStream`?

Reply:
taken care in Instance PR#619